app/lib/bank_details_pdf.rb

```diff
@@ -1,3 +1,5 @@
+require 'mini_magick'
+
```

Comment:
This looks unrelated, why was it included?

Reply:
Yes, it is unrelated. My test failed so I think it must be a failure that only happens when run in a certain order. Generally the suite passes. I probably should remove it and maybe we should research this separately. I am curious if that spec fails for you if you run it individually .

Reply:
Yes tests are passing for me